### ISSUE TITLE
Validation that does not stop the form from submitting?

### ISSUE THREAD

**Issue by precisionpete** [formvalidation/formvalidation#860]
I know this is not a small request. But can you see any way this tool could be used for a warning validation which does not prevent the form from being saved? 

Our app has some extensive forms. We do require some fields. When a user is just browsing, we do not require all of their profile fields to be complete. However, until certain profile fields are complete, we do not allow you all of the functionality of the app.

e.g. "Name" and "email" are required fields. "Skills" and "experience" will be required before you can apply for a position. 

Coding this in the app is not a problem. Representing it in the user interface is a challenge.

Basically, I need to be able to highlight certain fields as validation would normally do, preferably in a 3rd color (e.g. Green/Yellow/Red). But not have them prevent the form from being submitted.

Any ideas?
